public_html/administrator/components/com_jed/models/reviews.php

```diff
@@ -209,6 +217,12 @@ protected function getListQuery()
 			$query->where($db->quoteName('reviews.created_by') . ' = ' . (int) $reviewer);
 		}
 
+		$flagged = $this->getState('filter.flagged');
+		if (is_numeric($flagged))
```

Comment:
Empty line above the if, Joomla Codestyle. That goes for all filters by the look of it.